mteb/models/sentence_transformers_models.py

```diff
@@ -215,7 +215,13 @@
     use_instructions=False,
     superseded_by=None,
     adapted_from=None,
-    training_datasets=None,
+    training_datasets={
+        # CommonCrawl
+        # wiki  05-21-2020 dump
+        "WikipediaRetrievalMultilingual": [],
+        "WikipediaRerankingMultilingual": [],
```

Comment:
Since these are partly synthetic data I would not add them here (they are also not constructed the same way). We have generally not added it unless it is the same dataset construction (but I agree that that boundary is somewhat arbitrary).

Reply:
```suggestion
```